src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -113,6 +131,17 @@ public override bool Execute()
         GatherAotSourcesObjects(aotSources, aotObjects, extraSources, linkerArgs);
         GatherLinkerArgs(linkerArgs);
 
+        File.WriteAllText(Path.Combine(OutputDirectory, "library_builder.h"),
```

Comment:
I think we should follow delimitator used by other runtimes header, so library-builder.h